### ISSUE TITLE
Implement the user store

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#41]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

To have a low-level repository of user accounts.

### Why

So that the user manager class can persist user data in the Xperience database.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
